app/models/invite.rb

```diff
@@ -251,6 +251,23 @@ def resend_invite
     Jobs.enqueue(:invite_email, invite_id: self.id)
   end
 
+  def warnings(guardian)
+    return @warnings if !@warnings.nil?
```

Comment:
```suggestion
    return @warnings if @warnings.present?
```

Reply:
Your suggestion is not equivalent. No warnings (as in an empty `[]`) is a valid value and should be returned. Given how confusing this can be, I'll rewrite it a bit different.